src/page/sinfo.rs

```diff
@@ -47,6 +49,26 @@ impl From<Class> for SecInfo {
     }
 }
 
+/// Error codes for `SecInfo::accept()`
+#[derive(Debug)]
+pub enum AcceptError {
+    /// Unknown error
+    Unknown,
```

Comment:
I'm wondering should we just use `panic!()` for unknown case. This should never happen in a correctly working CPU.

Reply:
I'm okay with that. However, other comments:

1. The panic should have a clear error message like: `panic!("unknown EACCEPT error code")`.
2. I don't think we need two error types. Merge `AcceptError` and `AcceptCopyError`.
3. Enums should `#[derive(Copy, Clone, PartialEq, Eq)]`.